src/supertux/flip_level_transformer.hpp

```diff
@@ -33,20 +33,11 @@ class Decal;
 /** Vertically or horizontally flip a level */
 class FlipLevelTransformer final : public LevelTransformer
 {
+public:
+  static Flip transform_flip(Flip flip);
```

Comment:
Is there any upside to use this rather than taking the flip as (non-const) reference and modifying it directly that way?

(Suggesting that option as it would avoid an extra assignment in the code from callers)